qiita_pet/handlers/api_proxy/prep_template.py

```diff
@@ -0,0 +1,53 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+# This is the only folder in qiita_pet that should import outside qiita_pet
+# The idea is that this proxies the call and response dicts we expect from the
+# Qiita API once we build it. This will be removed and replaced with API calls
+# when the API is complete.
+from qiita_pet.handlers.api_proxy.util import check_access
+from qiita_db.study import Study
+
+
+def study_prep_proxy(study_id, user_id):
+    """Equivalent to GET request to '/study/(ID)/prep_template'
+
+    Parameters
+    ----------
+    study_id : int
+        Study id to get prep template info for
+    user_id : str
+        User id requesting the prep templates
+
+    Returns
+    -------
+    dict of list of dict
+        prep template information seperated by data type, in the form
```

Comment:
Coming really late to the party.

What do you guys think about substituting out this dict of list of dict data types with pandas?